flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/rules/logical/FlinkJoinToMultiJoinRuleTest.xml

```diff
@@ -548,6 +549,46 @@ LogicalProject(a=[$0], b=[$1], c=[$2], d=[$3])
    :  +- LogicalTableScan(table=[[default_catalog, default_database, T2, source: [TestTableSource(c, d)]]])
    +- LogicalProject(e=[$0])
       +- LogicalTableScan(table=[[default_catalog, default_database, T3, source: [TestTableSource(e, f)]]])
+]]>
+    </Resource>
+  </TestCase>
+  <TestCase name="testMultiLeftOuterJoinWithAllKeyInLeft">
+    <Resource name="sql">
+      <![CDATA[
+SELECT * FROM T1 LEFT OUTER JOIN T2 ON a = c LEFT OUTER JOIN 
+(SELECT * FROM T3) ON a = e LEFT OUTER JOIN
+(SELECT * FROM T4) ON a = g LEFT OUTER JOIN
+(SELECT * FROM T5) ON a = i
```

Comment:
the query can be simplied as:

SELECT * FROM T1 
LEFT OUTER JOIN T2 ON a = c 
LEFT OUTER JOIN T3 ON a = e
LEFT OUTER JOIN T4 ON a = g 
LEFT OUTER JOIN T5 ON a = i

Reply:
> the query can be simplied as:
> 
> SELECT * FROM T1 LEFT OUTER JOIN T2 ON a = c LEFT OUTER JOIN T3 ON a = e LEFT OUTER JOIN T4 ON a = g LEFT OUTER JOIN T5 ON a = i

Done!